plutus-tx/src/PlutusTx/Blueprint/Validator.hs

```diff
@@ -35,7 +35,7 @@ data ValidatorBlueprint (referencedTypes :: [Type]) = MkValidatorBlueprint
   -- ^ A description of the redeemer format expected by this validator.
   , validatorDatum        :: Maybe (ArgumentBlueprint referencedTypes)
   -- ^ A description of the datum format expected by this validator.
-  , validatorParameters   :: Maybe (NonEmpty (ParameterBlueprint referencedTypes))
+  , validatorParameters   :: [ParameterBlueprint referencedTypes]
```

Comment:
We discussed this, but indeed I think this is a case where the `Nothing` case is not meaningfully different. Having no parameters and having an empty list of parameters is the same.